detections/application/splunk_digital_certificates_infrastructure_version.yml

```diff
@@ -0,0 +1,51 @@
+name: Splunk Digital Certificates Infrastructure Version
+id: 3c162281-7edb-4ebc-b9a4-5087aaf28fa7
+version: 1
+date: '2022-05-26'
+author: Lou Stella, Splunk
+type: Hunting
+datamodel: []
+description: On June 14th, 2022, Splunk released a security advisory relating to TLS validation occuring between different parts of Splunk Enterprise Infrastructure. In addition to upgrading to Splunk Enterprise 9.0 or later, several configuration settings need to be set. This search will check those configurations on the search head it is run from as well as its search peers. Other components such as additional search heads or anything this rest command cannot be distributed to will need to be manually checked. 
```

Comment:
don't think the splunk disclosure comment is useful here - maybe just shorten to what the detection actually does...?